hedera-node/src/main/java/com/hedera/services/store/TopicStore.java

```diff
@@ -62,4 +70,46 @@ public void persistNew(Topic model) {
 		currentTopics.put(id, merkleTopic);
 		transactionRecordService.includeChangesToTopic(model);
 	}
+
+	/**
+	 * Returns a model of the requested topic, with operations that can be used to
+	 * implement business logic in a transaction.
+	 *
+	 * <b>IMPORTANT:</b> Changes to the returned model are not automatically persisted
+	 * to state! The altered model must be passed to {@link TopicStore#persistTopic(Topic)}
+	 * in order for its changes to be applied to the Swirlds state, and included in the
+	 * {@link com.hedera.services.state.submerkle.ExpirableTxnRecord} for the active transaction.
+	 *
+	 * @param id
+	 * 		the topic to load
+	 * @return a usable model of the topic
+	 * @throws InvalidTransactionException
+	 * 		if the requested topic is missing, deleted, or expired and pending removal
+	 */
+	public Topic loadTopic(Id id) {
+		final var merkleTopic = topics.get().get(EntityNum.fromLong(id.getNum()));
+		validateUsable(merkleTopic);
+		return fromMerkle(merkleTopic, id);
+	}
+
+	public boolean isAuthorized(Id id) {
```

Comment:
This method is not required and should be removed as explained in the TransitionLogic comment